### DISCUSSION TITLE
Update Clue Scroll Emotes

### DISCUSSION THREAD

**Discussion by Wexxorz** [runelite/runelite#16418] [Ideas]
As people are now playing runescape with efficiency in their head, lots of clue scroll emotes are far from where you can actually do them. sometime its only 1-2squares, but in other cases its 30+ squares , like the one in the picture.

![image](https://user-images.githubusercontent.com/128922695/227730493-724a5d4d-6ebf-464b-a877-20fd2d51f689.png)

**Comment by TheLope** [comment]
Bump for visibility

**Comment by TheLope** [comment]
Stash/emote steps: Most you can do right next to the stash unit but some require a few steps away. But current clues show spots to do the emote that are like 10 tiles away.
ex: Easy clue - Dance at fally party room. Current tile marked is inside next to the table where in reality you can dance outside the doors of the building next to stash unit. 

Overall we should have dig locations be near meta tp.

**Reply by Felanbird** [reply]
I've been thinking about this a lot because of how many clues I have ground markered right next to the STASH unit, but I figured it would be argued against due to being intrusive to users who don't use the STASH units/general noticing of the Clue markers changing spots.

My thought was to keep multiple sets of coords, the current ones and the STASH ones, but that idea did seem like a lot of effort

**Reply by geheur** [reply]
Adding my 2 cents: the plugin should remember where you last dug/emoted, or let you set a clue-scroll-plugin specific tile for your current clue.

Maximum customizability, and also adds a progression element of upgrading all the clue helpers.

**Reply by TheLope** [reply]
More customizability would always be good that sounds great

**Reply by DelVision** [reply]
perhaps you could do custom edit on the location with right click on the clue scroll overlay similar to how you can set notes, you could also have "Edit tile" when shift + right clicking the overlay which would let you move the default tile to where you want to perform the emote / dig

Also as an additional suggestion for editing clue helper: It would be nice if you could hide the massive clue icon on top of the objects you need to click on, single arrow pointing towards them is plenty enough as the clue scroll icon is often hiding the click box for no reason. (You can already do this in Quest Helper which had similar icon blocking the stuff you needed to click and without the icon it looks much more immersive and it's easier to click)

Like TheLope said, more customizability is always great and clue helper doesn't allow too almost any customizing currently (many of the clues also have way too much unnecessary text which would be neat if you could remove and purely use your own notes instead).

**Comment by TheLope** [comment]
Bump?
